### ISSUE TITLE
New Player and Bug Reporting note should direct players to this repo

### ISSUE THREAD

**Issue by Psypher9** [rezalas/riftshadow#170]
When starting as a new player there is a banner message that displays to show players how to report bugs. It seems that the original place to do that was at `http://forms.riftshadow.org`. 

That no longer seems available. Since this project seems to be alive here, it seems that bug reports should come here.

```bash

** The easiest way to report BUGS and TYPOS is on the Riftshadow forums.

All ideas are welcome, and criticisms equally so.  You can communicate
with the immortals and other players at http://forums.riftshadow.org.
```

Expected behavior:

- Guidance for new players on bug reporting should send players to this repo for issues or discussions.

**Comment by sean-gilliam** [comment]
This text is located in the `helpfiles` table. Should be easily fixed with an UPDATE command.

For reference:
```sql
select * from rift.helpfiles where title ='MOTD';
```

**Comment by sean-gilliam** [comment]
Closed by #183
